packages/atomic/src/utils/props-utils.ts

```diff
@@ -30,6 +38,56 @@ export function MapProp(opts?: MapPropOptions) {
   };
 }
 
+export function ArrayProp(opts: ArrayPropOptions) {
+  const logWithDeprecation = (msg: string, ...other: unknown[]) =>
+    opts.deprecationWarning
+      ? console.warn(
+          `${msg} This will be enforced in the next major version`,
+          other
+        )
+      : console.error(msg, other);
+
+  return (component: ComponentInterface, variableName: string) => {
+    const {componentWillLoad} = component;
+    const attributeWithBackets = camelToKebab(variableName);
```

Comment:
Did you mean to rename this? There's no more brackets.

Reply:
I removed the bracket annotation, yes